### ISSUE TITLE
[BUG] Image directory caching from a remote SSH server

### ISSUE THREAD

**Issue by JonasVerbickas** [cweijan/vscode-office#264]
First of all you have built a great extension. 

However, there is an issue when working with large image directories on remote servers via SSH.
Whenever I click on an image within a directory containing tens of thousands of images it will load for a LONG time. I'm assuming the extension attempts to download the entire directory of images to the local machine in order to display them in gallery view and allow to switch between images quickly. This creates another issue, from what I can tell, images are being cached by the extension, therefore, after opening a few large directories my MacBook storage quickly fills up even though I open the directories only a few times a week at best. My theory is further supported by the fact that whenever I would disable an extension and click on an image to view it in the native VSCode image editor there is no long waiting time and I can see the image almost instantly.

I presume this could be easily fixed by setting a maximum number of images that would be downloaded from a directory when using SSH e.g. limiting the number of downloaded images to 100 and only after 101st image is selected downloading another 100.

- OS: MacOS Ventura local / Remote Ubuntu 22.04
- Extension Version: v3.1.7

**Comment by cweijan** [comment]
Hi, this issue has been fixed. The default image opening method in VS Code will no longer be changed.
